packages/mask/src/plugins/Collectible/src/SNSAdaptor/Shared/DetailsCard.tsx

```diff
@@ -84,7 +84,9 @@ export function DetailsCard(props: DetailsCardProps) {
         { title: t('token_standard'), value: Others?.formatSchemaType(asset.schema || asset.contract?.schema) },
         {
             title: t('plugin_collectible_creator_earning'),
-            value: `${Number.parseInt(asset.contract?.creatorEarning || '0', 10) / 100}%` ?? '0',
+            value: asset.contract?.creatorEarning
+                ? `${Number.parseInt(asset.contract.creatorEarning, 10) / 100}%`
```

Comment:
`formatPercentage`

Reply:
The calculating is different from `formatPercentage`, beside I didn't figure out what's the value of `asset.contract.creatorEarning`.